tools/workbench/src/main/java/org/eclipse/rdf4j/workbench/util/WorkbenchRequest.java

```diff
@@ -116,7 +116,7 @@ public int getInt(String name) throws BadRequestException {
 
 	@Override
 	public String getParameter(String name) {
-		String result = null;
+		String result;
```

Comment:
Interesting that this can now be done - in older versions of Java this would result in a "value may have not have been initialized" compile error. Which version of Java was this changed in? I assume the behavior is now that it implicitly initializes local vars to `null`?

Reply:
Don't know how new of a feature this is. The compiler just checks that there is no code path where an unitialized variable can be read.

Useful for things like:

```
int res;
if(something()){
    res = 1;
}else{
    res = 7;
}


```